### PR TITLE
task(rn) removes webviews and opens urls in browser

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: react/features/settings/components/native/SettingsView.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
     Alert,
+    Linking,
     NativeModules,
     Platform,
     ScrollView,
     Text,
     View
 } from 'react-native';
-import { Divider } from 'react-native-paper';
+import { Button, Divider } from 'react-native-paper';
```
[the remark this answers is not in the document]
Comment (saghul) [reply react/features/settings/components/native/SettingsView.tsx:15]: Make it tertiary, IIRC they show as a link (visually)

**Review by saghul — CHANGES_REQUESTED** [review]

File: react/features/settings/components/native/SettingsView.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Calinteodor) [reply react/features/settings/components/native/SettingsView.tsx]: image

The links don't have the blue token color, but INMHO, I think they look better now. WDYT @saghul ?

[the remark this answers is not in the document]
Comment (Calinteodor) [reply react/features/settings/components/native/SettingsView.tsx]: Aside from that, everything works and feels as expected.

[the remark this answers is not in the document]
Comment (saghul) [reply react/features/settings/components/native/SettingsView.tsx]: LGTM, they are tertiary buttons and that's the design we got for them :-)

**Review by Calinteodor — APPROVED** [review]

**Review by saghul — APPROVED** [review]

**Pull request merged**